resources/function_help/json/is_selected

```diff
@@ -14,9 +14,9 @@
             "variant": "One 'feature' parameter",
             "variant_description": "If called with a 'feature' parameter only, the function returns true if the specified feature from the current layer is selected.",
             "arguments": [ { "arg": "feature", "description": "The feature which should be checked for selection." } ],
-            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if the current atlas feature is selected." },
-                          { "expression": "is_selected(get_feature('streets', 'name', 'Main St.')))", "returns": "True if the unique named \"Main St.\" feature on the active \"streets\" layer is selected." },
-                          { "expression": "is_selected(get_feature_by_id('streets', 1))", "returns": "True if the feature with the id 1 on the active \"streets\" layer is selected." }
+            "examples": [ { "expression": "is_selected(@atlas_feature)", "returns": "True if a selected feature on the current layer is the active atlas feature." },
```

Comment:
I suggest chaging 'on the current' to 'of the current'. 'on' doesn't make much sense to me in this phrase.